### ISSUE TITLE
Failed resolve plugin

### ISSUE THREAD

**Issue by dekelmos** [sksamuel/sbt-avro4s#21]
Tried using the plugin on my own project and got the following error. same error appeared when tried example project [avro-scala-demo](https://github.com/jkpl/avro-scala-demo)
sbt.librarymanagement.ResolveException: unresolved dependency: com.sksamuel.avro4s#sbt-avro4s;1.0.0: not found

**Comment by wtanner** [comment]
I don't believe sbt 1.0 is supported yet (https://github.com/sksamuel/sbt-avro4s/issues/16), which might be the source of your error.
